src/platform/intel/cavs/lib/clk.c

```diff
@@ -118,8 +110,24 @@ void platform_clock_on_waiti(void)
 		set_cpu_current_freq_idx(CPU_LPRO_FREQ_IDX);
 	}
 }
+
+static inline void clock_platform_update_active_freq_idx(int freq_idx)
+{
+	*cache_to_uncache(&active_freq_idx) = freq_idx;
```

Comment:
I know this is copied from above, but can we mke this assignment cleaner.  e.g.
```c
int *uncached_freq_idx = cache_to_uncache();
*uncached_freq_idx = freq_idx;
```

Reply:
updated